mscore/musescore.cpp

```diff
@@ -4140,7 +4140,10 @@ void MuseScore::changeState(ScoreState val)
       if (getAction("split-measure")->isEnabled())
             getAction("split-measure")->setEnabled(cs && cs->masterScore()->excerpts().size() == 0);
 
-      getAction("edit-toolbars")->setEnabled(!Workspace::currentWorkspace->readOnly());
+      Workspace* currentWorkspace = Workspace::currentWorkspace;
+      if (currentWorkspace) {
```

Comment:
When running tst_runscripts test the currentWorkspace returns nullptr and MuseScore crashes before it even gets to run any of the tests. I've put in an explicit check to avoid this. It now passes 43/44 tests, which is exactly what master is currently passing. Is currentWorkspace unexpectedly being null because the tests are running MuseScore in "headless" mode (if that's the right terminology)? Might this be an issue elsewhere?